net-libs/libwebsockets/libwebsockets-4.0.4.ebuild

```diff
@@ -12,14 +12,16 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="MIT"
 SLOT="0/16" # libwebsockets.so.16
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
-	+lejp libev libevent libressl libuv peer-limits server-status smtp socks5
-	sqlite3 ssl static-libs threads zip"
+IUSE="access-log caps cgi client dbus +dir +extensions +file-ops generic-sessions http-proxy +http-basic-auth http2 ipv6
+	+lejp libev libevent libressl libuv mbedtls peer-limits server-status smtp socks5
+	sqlite3 ssl static-libs +testapps threads zip"
 
 REQUIRED_USE="
```

Comment:
In line with my USE comments, if you can trim down REQUIRED_USE at all, that'd be nice. It can be a bit frustrating for users when they have to work out a whole graph. Not a requirement though.

Reply:
And I'd rather have it err out at graph generation time than build time.

Reply:
Agreed. Just do not want useless entries in there.

Reply:
Yes, as @jkroonza said these will cause build failures, hence they are included in REQUIRED_USE